test/FastTests/Server/Documents/Revisions/RevisionsTests.cs

```diff
@@ -1087,6 +1087,86 @@ public async Task DeleteRevisionsWhenNoneExistShouldNotThrow()
             }
         }
 
+        [Fact]
+        public async Task CollectionCaseSensitiveTest1()
+        {
+            using (var store = GetDocumentStore())
+            {
+                var id = "user/1";
+                var configuration = new RevisionsConfiguration
+                {
+                    Collections = new Dictionary<string, RevisionsCollectionConfiguration>
+                    {
+                        ["uSErs"] = new RevisionsCollectionConfiguration { Disabled = false }
```

Comment:
What will happen if I will put `users` and `USERS` on the Client side and push it to the server (can we have a test for that)?

Reply:
Yes. I will add one